scripts/tank_cmd.bat

```diff
@@ -1,13 +1,13 @@
 @echo off
 
 rem  Copyright (c) 2013 Shotgun Software Inc.
-rem
+rem  
```

Comment:
The whitespace removal caused issues when upgrading tk-core, which would replace the current copy with extra whitespaces with the one that didn't. Since the .bat processor reads one unbuffered line at a time from disk, it means the content of the file has shifted "left" after the update and it starts reading from the middle of a line. So we're putting those back in for now.